src/test/common/steps/terms-and-conditions-steps.js

```diff
@@ -0,0 +1,44 @@
+const { Given, When, Then } = require('cucumber')
+
+const pages = require('./pages')
+const { assertErrorHeaderTextPresent, assertFieldErrorAndLinkTextPresentAndCorrect, setupWiremockMappingsWithStatus } = require('./common-steps')
+
+async function acceptTsAndCs () {
+  await pages.termsAndConditions.openDirect(pages.url)
+  await pages.termsAndConditions.selectCheckbox()
+}
+
+async function acceptTsAndCsAndSubmitApplication () {
+  await acceptTsAndCs()
+  await pages.termsAndConditions.submitForm()
+}
+
+Given(/^I accept the terms and conditions$/, async function () {
+  await acceptTsAndCs()
+})
+
+When(/^I accept the terms and conditions and submit my application$/, async function () {
+  await setupWiremockMappingsWithStatus('ELIGIBLE')
+  await acceptTsAndCsAndSubmitApplication()
+})
+
+When(/^I click continue without accepting the terms and conditions$/, async function () {
+  await pages.termsAndConditions.openDirect(pages.url)
+  await pages.termsAndConditions.submitForm()
+})
+
+Then('I am told I must accept the terms and conditions', async function () {
+  await assertErrorHeaderTextPresent(pages.termsAndConditions)
+  await assertFieldErrorAndLinkTextPresentAndCorrect(
+    'agree-error',
```

Comment:
These ids should be stored in the terms and conditions page object rather than in the steps as per the other pages.

Reply:
done